src/UIShell/Content.svelte

```diff
@@ -11,14 +11,14 @@
    * We manually unset the left margin if the `SideNav`
    * is collapsed and if it's not the `rail` variant.
    */
-  $: unsetLeftMargin = $isSideNavCollapsed && !$isSideNavRail;
+  $: unsetLeftMargin = !$isSideNavRail && !$isSideNavExpanded;
 </script>
 
 <main
   id="{id}"
   class:bx--content="{true}"
+  style="{unsetLeftMargin ? 'margin-left: 0' : ''}"
   {...$$restProps}
-  style="{unsetLeftMargin ? 'margin-left: 0;' : ''} {$$restProps.style}"
```

Comment:
I don't think that this should be changed since `$$restProps.style` will override the attribute.